src/components/MarkdownProvider/components/CodeExample/utils/retrieveCodesandboxParameters.ts

```diff
@@ -61,7 +61,7 @@ const packageJson = `
     "@zendeskgarden/react-buttons": "^8.0.0",
     "@zendeskgarden/react-chrome": "^8.0.0",
     "@zendeskgarden/react-dropdowns": "^8.0.0",
-    "@zendeskgarden/react-grid": "^8.0.0",
+    "@zendeskgarden/react-grid": "latest",
```

Comment:
Are we sure this is something we want to do? I'm concerned about selecting a semver that supports the mental mapping for developers looking at these examples in-depth. @austin94 thoughts?

Reply:
It seems that Codesandbox doesn't map to the latest valid semver range when `^8.0.0` is used. `latest` is common in other repos that use this functionality so it may be our only options.

Reply:
Maybe we should move all Garden dependencies in this to use `latest`?

Reply:
Yeah, at this point I would vote for consistency. Although, we're going to have to keep this in mind when moving to a new major version because the old version examples will tend to break.

Reply:
Moving all of them to the latest was my initial inclination. I can move them all to latest, and keep an eye out for major releases of `react-components` 👍.

Reply:
Another option would be to bump all the minors. `^8.9.0` Then this won't automatically break back-versions when we publish a v9.0 of the website.

Reply:
btw, I don't think this has been called out. But whatever is happening here is definitely a hackaround for a Codesandbox bug. Here's the latest dependency list with `^8.0.0`:

<img width="351" alt="Screen Shot 2020-05-20 at 2 06 21 PM" src="https://user-images.githubusercontent.com/143773/82497663-7fa3a800-9aa3-11ea-8b5a-8333e6f39bfb.png">

And then with `latest`:

<img width="346" alt="Screen Shot 2020-05-20 at 2 07 59 PM" src="https://user-images.githubusercontent.com/143773/82497687-89c5a680-9aa3-11ea-8ca3-d60ac200467e.png">

Both are showing the use of version `8.10.0`. But only the `latest` supports recent grid props, etc.

Reply:
Seems to be a cache fault with CodeSandbox. When I change the grid range to `^8.x` everything seems to work. For now, I'd recommend setting all of these to `^8.x`.

<img width="348" alt="Screen Shot 2020-05-20 at 2 18 57 PM" src="https://user-images.githubusercontent.com/143773/82498475-e8d7eb00-9aa4-11ea-92e8-75d861831b3b.png">

Reply:
Thanks, @jzempel! I'll update the react-component deps to ^8.x.